packages/connect-web-test/src/fetch.spec.ts

```diff
@@ -1,176 +0,0 @@
-// Copyright 2021-2024 The Connect Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-import { TestService } from "./gen/connectrpc/conformance/v1/test_connect.js";
-import {
-  SimpleRequest,
-  SimpleResponse,
-} from "./gen/connectrpc/conformance/v1/messages_pb.js";
-import {
-  createConnectTransport,
-  createGrpcWebTransport,
-} from "@connectrpc/connect-web";
-
-describe("custom fetch", function () {
```

Comment:
Custom fetch for SSR has some peculiar requirements, and it would be bad if we regress on them by accident. So we need to keep this.